### DISCUSSION TITLE
Issue connecting to Athena

### DISCUSSION THREAD

**Discussion by zhsiao925** [apache/superset#24349] [Q&A / Help]
I am attempting to connect to Amazon Athena using PyAthena Rest, but I keep encountering this error. I have completed the installation on to a linux instance through docker, and have installed the recommended packages. How can I fix this exception?

superset.databases.commands.exceptions.DatabaseTestConnectionUnexpectedError: [SupersetError(message="type object 'AthenaRestDialect' has no attribute 'dbapi'", error_type=<SupersetErrorType.GENERIC_DB_ENGINE_ERROR: 'GENERIC_DB_ENGINE_ERROR'>, level=<ErrorLevel.ERROR: 'error'>, extra={'engine_name': 'Amazon Athena', 'issue_codes': [{'code': 1002, 'message': 'Issue 1002 - The database returned an unexpected error.'}]})]

**Comment by Flo451** [comment]
Relates to: https://github.com/laughingman7743/PyAthena/issues/447

For me downgrading PyAthena to versions before 3.0.3 helped.
